test/unit/console_log_utils_test.dart

```diff
@@ -131,17 +131,50 @@ void main() {
         expect(printCalls, ['foo', 'bar']);
       });
 
-      test('prints logs that use formatter syntax', () {
-        // This also tests functionally that the print call occurs in the right zone,
-        // which need to happen for them to be forwarded in the terminal in a test environment.
-        // If it wasn't in the right zone, we wouldn't be able to record it, and neither would
-        // the test package.
-        final printCalls = recordPrintCalls(() {
-          printConsoleLogs(() {
-            callMethod(getProperty(window, 'console'), 'log', ['%s World Number %d! %j', 'Hello', 5, jsify({'doWeComeInPeace':false}), 'additional']);
+      group('prints logs that use formatter syntax', () {
+        test('with exact amount of args', () {
+          // This also tests functionally that the print call occurs in the right zone,
+          // which need to happen for them to be forwarded in the terminal in a test environment.
+          // If it wasn't in the right zone, we wouldn't be able to record it, and neither would
+          // the test package.
+          final printCalls = recordPrintCalls(() {
+            printConsoleLogs(() {
+              callMethod(getProperty(window, 'console'), 'log', ['%s World Number %d! %j', 'Hello']);
```

Comment:
Looks like this is missing the jsified argument, the number, and the additional arg